src/components/editTestCase/qdm/EditTestCase.tsx

```diff
@@ -61,8 +84,25 @@ const EditTestCase = () => {
     }
   };
 
+  const submitForm = () => {
+    const submitTestCase = {
+      ...currentTestCase,
+      json: testCaseJson,
+    };
+
+    testCaseService.current
+      .updateTestCase(submitTestCase, measureId)
```

Comment:
I believe we should also update measureStore to keep DB and measureStore in Sync

Reply:
Talked with Rohit will do the merge now and do the update measure store next.